common/scripts/lib/_helpers.sh

```diff
@@ -50,6 +50,14 @@ __check_os_and_architecture() {
   source "$ADMIRAL_ENV"
 }
 
+__set_installed_docker_version() {
+  if command -v "docker" &>/dev/null; then
```

Comment:
will thsi not fail if docker is not installed?

Reply:
this checks if `docker` is command is present before calling `docker version` [reference](https://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script)